### PR TITLE
Update db sql  (added inputs, removed the exteneLinks table)

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

**Review by GeorgesFORTRY — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE!

File: index.js [hunk 1]
```diff
@@ -110,6 +111,40 @@ app.get('/external_links', async (req, res) => {
...
+    pass: process.env.EMAIL_PASSWORD,
+  },
+});
+
+app.post('/contact', (req, res) => {
+  const { firstname } = req.body;
+  const { lastname } = req.body;
+  const { society } = req.body;
+  const { email } = req.body;
+  const { message } = req.body;
```
Comment (LoicBrassart) [inline index.js:163]: `consy {var1, var2, var3} = req.body`

**Review by LoicBrassart — APPROVED** [review]
Minor replacement

**Pull request merged**